src/matlab/ctfunctions.cpp

```diff
@@ -81,6 +81,16 @@ void ctfunctions(int nlhs, mxArray* plhs[],
         iok = clear_rxnpath();
         break;
 
+        // get string of data directories
+    case 5:
+        sep = getString(prhs[2]);
+        buflen = getDataDirectories(buflen, output_buf, sep);
```

Comment:
I think it is more clear to just put 0 in place of `buflen` and `output_buf` in this initial call.